cosmos/cache.py

```diff
@@ -400,3 +407,60 @@ def create_cache_profile(version: str, profile_content: str) -> Path:
     profile_yml_path = profile_yml_dir / DEFAULT_PROFILES_FILE_NAME
     profile_yml_path.write_text(profile_content)
     return profile_yml_path
+
+
+def is_cache_lockfile_enabled(project_dir: Path) -> bool:
```

Comment:
BTW: it may be worth to rename any occurrence of `lockfile` to `package_lockfile` - Cosmos has other lockfiles (we're introducing one in the #1079

Reply:
Updated in commit d5b3e9b7bc9063a7a44c0e583bf4e79f8f51fe09 PTAL